src/components/course/data/service.js

```diff
@@ -130,4 +141,29 @@ export default class CourseService {
     const url = `${this.config.LICENSE_MANAGER_URL}/api/v1/license-subsidy/?${qs.stringify(options)}`;
     return this.cachedAuthenticatedHttpClient.get(url);
   }
+
+  fetchUserOfferSubsidy({ offers, catalogList }) {
+    // TODO this probably should be a fetch but we are using prefetch offer data here
```

Comment:
Maybe add to this TODO comment here to mention that we're missing the bit about discovering whether the user has an offer that applies to the specific course; that we're only naively checking that the user has an offer that applies to _any_ of the Enterprise's catalog(s).

I believe you're planning to file a ticket for this, but probably good to call it out in code too? :)

Reply:
sure! please check updated jsdoc on this method. I will file the ticket for this missing piece of course specificity

Reply:
ticket filed: https://openedx.atlassian.net/browse/ENT-3991